components/floatingactionbutton/stories/template.js

```diff
@@ -18,6 +20,8 @@ export const Template = ({
 		<button
 			class=${classMap({
 				[rootClass]: true,
+				"is-focused": isFocused,
```

Comment:
Floating action button does not have an is-focused class, so this arg and control should not be needed.